src/sql/workbench/api/browser/mainThreadAria.ts

```diff
@@ -0,0 +1,27 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { SqlMainContext, MainThreadAriaShape } from 'sql/workbench/api/common/sqlExtHost.protocol';
+import { extHostNamedCustomer } from 'vs/workbench/api/common/extHostCustomers';
+import { Disposable } from 'vs/base/common/lifecycle';
+import * as aria from 'vs/base/browser/ui/aria/aria';
+
+
+@extHostNamedCustomer(SqlMainContext.MainThreadAria)
+export class MainThreadAria extends Disposable implements MainThreadAriaShape {
```

Comment:
There's no way to avoid duplicating the aria shape between MainThread and ExtHost?